### PR TITLE
Edit Project2

### PR DESCRIPTION
**Pull request by zeizyy**
Modify Docker Compose section and add information about fixture.

### CODE REVIEW

File: Project2.md [hunk 1]
```diff
@@ -58,13 +54,68 @@ but I think it's a reasonable short-cut for a class project. The risk
...
+Compose is a tool for defining the topology and running multi-container Docker applications. 
+With Compose, you use a Compose file to configure your application’s services.
+Recall how we start web, mysql containers one by one in the previous project. This
+approach works fine when we have only two containers and the configuration is simple, but
+is certainly not scalable. Using Docker Compose, we will be able to stand up our containers
+using a single command. Here is a tutorial about how Docker Compose works:
+
+First, install Docker Compose by doing
+
+	$ curl -L https://github.com/docker/compose/releases/download/1.8.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```
Comment (thomaspinckney3) [inline Project2.md]: Do you need to install Compose separately or is it part of Docker for Mac/Windows?

File: Project2.md [hunk 2]
```diff
@@ -58,13 +54,68 @@ but I think it's a reasonable short-cut for a class project. The risk
...
+	$ chmod +x /usr/local/bin/docker-compose
+
+
+When you've got Compose installed, create a new directory with
+
+	mkdir Compose
```
Comment (thomaspinckney3) [inline Project2.md]: Clarify where in their code hierarchy this directory should be created.

File: Project2.md [hunk 3]
```diff
@@ -58,13 +54,68 @@ but I think it's a reasonable short-cut for a class project. The risk
...
+	cd Compose
+
+Then go ahead and create a new file, docker-compose.yml, with your favorite text editor. Docker Compose uses these files, called YAML files, to automatically set up or start a group of containers. This is handy, much easier than managing each container individually. You can specify a lot of different options here, but for now we're merely going to create our entity layer and then link it to our mysql database. You need to make sure the mysql container we created from last time is started.
+
+You can tell compose to create a new container by giving it a name and an image to use.
+
+	models:
+	    image: tp33/django:1.3
```
Comment (thomaspinckney3) [inline Project2.md]: I don't believe that version on the container is correct. Note that in project 1 there are no versions.

File: Project2.md [hunk 4]
```diff
@@ -58,13 +54,68 @@ but I think it's a reasonable short-cut for a class project. The risk
...
+	    external_links:
+	      -  mysql:db
+	    volumes:
+	      - <your_file_path>:/app
+	    ports:
+	      - "8001:8000"
+	    command: bash -c "mod_wsgi-express start-server --reload-on-changes stuff-models/wsgi.py"
```
Comment (thomaspinckney3) [inline Project2.md]: Clarify that stuff is just an example project name.

File: Project2.md [hunk 5]
```diff
@@ -159,12 +209,34 @@ you'll need to use your Django manage.py to generate a set of SQL commands neede
 your new models. Then you can apply these commands to make them actually take affect. Django breaks this into two
 stages so that you can check the commands into git on your dev machine and then later apply them to many different db's -- in theory you might have many dev db instances, some testing/qa instances and then prod db instances. See the Django getting started or model documentation for more on migrations and how to use them.
 
+Fixtures
+--------
+Final thing before we call it a day, you need to add fixtures so that we are provided with initial data when grading your project.
```
Comment (thomaspinckney3) [inline Project2.md]: Mention that fixtures aren't just about making grading easier, they're about making any sort of testing easier.

**Comment by zeizyy** [comment]
Updated

**Pull request merged**